src/vector.js

```diff
@@ -346,18 +295,25 @@ class Vector {
 
       if (a.length === b.length && a.length === 3) {
         let params = [
-          a[1] * b[2] - a[2] * b[1],
```

Comment:
This was cleaner and simpler

Reply:
The previous implementation introduces rounding errors.